systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -77,8 +74,7 @@ public abstract class AbstractST implements TestSeparator {
     protected final ResourceManager resourceManager = ResourceManager.getInstance();
     protected final TestSuiteNamespaceManager testSuiteNamespaceManager = TestSuiteNamespaceManager.getInstance();
     private final SuiteThreadController parallelSuiteController = SuiteThreadController.getInstance();
-    protected SetupClusterOperator clusterOperator;
-    protected OlmResource olmResource;
+    protected SetupClusterOperator clusterOperator = SetupClusterOperator.getInstanceHolder();
```

Comment:
Wouldn't be better to rename `getInstanceHolder` to `getInstance`?

Reply:
yea sure